### PR TITLE
Feat/markercluster

### PR DESCRIPTION
**Pull request by mikemangas**
- added feature: when a certain zoom level is beeing reached, the markers (location symbols) are beeing bundeled /clustered, into 1 marker.

### CODE REVIEW

**Review by johnxgaehner — APPROVED** [review]
Hi Mike! it works.
just noticed, that you have a green background on the blue pins as well but nothing to add to the function. checked on desktop + mobile phone, both work fine!

**Review by namirsab — APPROVED** [review]
Hey @mikemangas impressive work!
Just a  suggestion and a question :) 
Keep it up! 💪

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -55,22 +56,24 @@ export default function Home() {
           url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
```
[the remark this answers is not in the document]
Comment (mikemangas) [reply client/src/pages/Home.js:56]: Thanks for the hint! I still need to figure out how to use this tile!!!
----
{s} means one of the available subdomains (used sequentially to help with browser parallel requests per domain limitation; subdomain values are specified in options; a, b or c by default, can be omitted), {z} — zoom level, {x} and {y} — tile coordinates. {r} can be used to add "@2x" to the URL to load retina tiles.

------

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -55,22 +56,24 @@ export default function Home() {
...
+                positionData?.geometry?.coordinates[0],
+              ]}
+            >
+              <Popup>
+                <>
+                  <button>CHECK-IN</button>
+                  <p>{positionData?.properties?.name}</p>
+                </>
+              </Popup>
+            </Marker>
```
[the remark this answers is not in the document]
Comment (mikemangas) [reply client/src/pages/Home.js:74]: thanks for the tipp! I actually thought about improving the readability of my code in a bit latter stage of the project. I hope thats fine :)

**Comment by mikemangas** [comment]
> Hi Mike! it works.
> just noticed, that you have a green background on the blue pins as well but nothing to add to the function. checked on desktop + mobile phone, both work fine!

Thanks for the review John! I just added a background color to the clusters - otherwise you would only see the number of markers, which looks way too anaesthetic. 

I will do the style part at the end of my project, i guess :)

**Pull request merged**